docs/src/index.md

```diff
@@ -75,9 +75,120 @@ julia> c = CalOpts(; kwargs...)
 - `poll_period`: the delay, in localticks, between successive control updates
 - `control_delay`: the delay, in localticks, at a node between making a measurement
    and the frequency update
+- `errors`: a list of the uncontrolled frequency of the oscillators
+- `base_freq`: used to scale the integrator in the controller. See the example below.
    
-   
-   
+      
+## Example
+
+The following is a more complete example. In particular, here we are setting the uncontrolled frequencies of the system to be approximately 200MHz, with a maximum error of 100ppm. 
+
+We are also running the system in units of nanoseconds, so 200MHz corresponds to a frequency of 0.2.  The sampling period of the controller is given by `poll_period`, but this is in units of localticks. Because sampling is driven by the local clock, it can only be set in localticks. The variable `base_freq` contains the approximate frequency (or *base frequency*) of the oscillators. Here we scale `poll_period` by the base frequency so that the sampling rate is about 1e5 nanoseconds.
```

Comment:
> so 200MHz corresponds to a frequency of 0.2.
Ideally the example would have line numbers in the code and we could refer to those lines explicitly. The poor's man replacement would be to refer to the variable definition (`base_freq` in this case). I believe it would make it more clear.